saleor/graphql/channel/filters.py

```diff
@@ -1,14 +1,6 @@
 from typing import Dict
 
-from graphql.error import GraphQLError
-
-LACK_OF_CHANNEL_IN_FILTERING_MSG = (
-    "You must provide a `channel` filter parameter to properly filter data."
-)
-
 
 def get_channel_slug_from_filter_data(filter_data: Dict):
-    channel_slug = filter_data.get("channel", None)
-    if not channel_slug:
-        raise GraphQLError(LACK_OF_CHANNEL_IN_FILTERING_MSG)
+    channel_slug = str(filter_data.get("channel", None))
```

Comment:
There is no chance that this situation appears because if a channel is not given it's replaced by default channel with the use of `get_default_channel_slug_or_graphql_error`. But maybe it should be changed to:
`channel_slug = str(filter_data["channel"])` which will be suggesting that `filter_data` always has a `channel` field.

Reply:
Ok, i've missed where the `filter_data` comes from. Looks fine then 👍🏻